### DISCUSSION TITLE
[QPROF] UI issue with get_cpu_time

### DISCUSSION THREAD

**Discussion by yakirgb** [vertica/VerticaPy#1314] [Potential bug] [answered]
Hi,
Using version `'1.1.0-beta-1—10102024-dc6dcae061e61c0dfee1e82dac295b4d4df24f19'`

Doing:
```
qprof_interface_1.get_cpu_time()
```
 `highchart`s looks good, i see the value + node name + path line.
![image](https://github.com/user-attachments/assets/9cae2d0f-6090-403b-86b3-04f06156c70a)

On  `matplotlib` , it's not possible to see the real value or the node name when clicking on the graph, the graph just not clickable.
![image](https://github.com/user-attachments/assets/f2eb95b3-6da8-4440-82b6-c7e89e209c46)
![image](https://github.com/user-attachments/assets/fb3dfd7d-967c-468f-b2b3-80cb72019ee0)

On `plotly` and `bar` the number very confusing because path id together with the real value and node name missing
![image](https://github.com/user-attachments/assets/12359be6-8627-4781-b6fc-447a4a782f7d)

Thank you, Yakir

**Comment by oualib** [comment, marked as answer]
I've created an issue for that one: https://github.com/vertica/VerticaPy/issues/1327

I'll close this discussion.

**Comment by oualib** [comment]
I tried to reproduce the experiment and it does not seem to be a bug:

**Highcharts:**

<img width="813" alt="Screenshot 2024-10-27 at 07 40 44" src="https://github.com/user-attachments/assets/5da46bfe-1497-4af9-a373-3adc2092895c">

**Plotly:**

<img width="806" alt="Screenshot 2024-10-27 at 07 40 55" src="https://github.com/user-attachments/assets/b9cc96cc-da70-4015-9546-8e040f97f3b5">

**Matplotlib:**

<img width="822" alt="Screenshot 2024-10-27 at 07 40 36" src="https://github.com/user-attachments/assets/9a058112-2ee5-4fcd-a9bf-189c87ed841f">

It seems that in your case, the graphic might be too large, or your Plotly version could be buggy. I'm using Plotly 5.6.0, and you can try adjusting the graphic size for better results. Each plot is an object that you can modify using the appropriate methods.

Ex:
```python
fig = obj.get_cpu_time()
fig.update_layout(width=800)
```
